### PR TITLE
Added check restart load test scenario

### PR DESCRIPTION
**Pull request by swapnaravula**
This branch have the below load test scenario, which will get the teacher names from the database and populates 01-teacherLoginsForRestart.csv before running the Restart-Check thread group.

staff-restart-pupil-check-use-db.jmx

### CODE REVIEW

File: load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+          <stringProp name="queryTimeout"></stringProp>
+          <stringProp name="resultSetHandler">Store as String</stringProp>
+        </JDBCSampler>
+        <hashTree/>
+        <JSR223PostProcessor guiclass="TestBeanGUI" testclass="JSR223PostProcessor" testname="JSR223 PostProcessor - Export Results To CSV" enabled="true">
+          <stringProp name="scriptLanguage">groovy</stringProp>
+          <stringProp name="parameters"></stringProp>
+          <stringProp name="filename"></stringProp>
+          <stringProp name="cacheKey">true</stringProp>
+          <stringProp name="script">resultSet = vars.getObject(&quot;resultSet&quot;)
```
Comment (GuyHarwood) [inline load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx]: what is this script for?
is it possible to add comments to this to explain what its for?

File: load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+      <hashTree>
+        <JDBCSampler guiclass="TestBeanGUI" testclass="JDBCSampler" testname="JDBC Request - Select Teacher Credentials" enabled="true">
+          <stringProp name="dataSource">mtc-admin</stringProp>
+          <stringProp name="queryType">Select Statement</stringProp>
+          <stringProp name="query">SELECT DISTINCT u.identifier as [userName], LEN(u.identifier) as [length]
+FROM mtc_admin.vewPupilsEligibleForRestart vpr
+INNER JOIN  mtc_admin.[user] u ON vpr.school_id = u.school_id
+ORDER BY LEN(u.identifier), u.identifier
+OFFSET ${OFFSET_VALUE_NODE} ROWS
+FETCH NEXT 1000 ROWS ONLY</stringProp>
```
Comment (GuyHarwood) [inline load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx]: same for this - is there a way to add comments / description to explain what its doing and why?

File: load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx (hunk 1 above)
Comment (swapnaravula) [reply load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx]: Added comments inside the script

File: load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx (hunk 2 above)
Comment (swapnaravula) [reply load-test/scenarios/restart-check/staff-restart-pupil-check-use-db.jmx]: Added comments as suggested.

**Pull request merged**